build/SignConfig.xml

```diff
@@ -3,12 +3,12 @@
   <job platform="" configuration="" dest="__INPATHROOT__" jobname="dep.controls" approvers="">
     <!-- Overwrite the input files to simplify downstream things like nuget and framework package generation. -->
     <file src="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" signType="WindowsRT" dest="__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" />
```

Comment:
>__INPATHROOT__\Release\x86\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll" [](start = 15, length = 81)

so in-place signing works?

Reply:
It seems to, I'm trying to verify that the nupkg and its contents are actually signed currently.

Reply:
Verification was completed.